web-ui/public/styles.css

```
@@ -0,0 +1,1129 @@
+  margin-top: 5px;
+}
+
+.chat-form .form-message:empty { display: none; }
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)